code/src/index.css

```diff
@@ -5,9 +5,212 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url('./assets/sparc-bg.jpg');
+  background-size: cover;
+  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* header and header animation  */
+
+h1 { 
+  font-size: 75px;
+  width: fit-content;
+  margin-left: 30%;
+  margin-right: 30%;
+  padding: 10px;
+  color: rgba(255, 255, 255, 0.829);
+  font-family: 'Quantico', sans-serif;
+  text-align: center;
+  border: 2px solid #fff;
+  border-radius: 2rem;
+  text-transform: uppercase;
+  animation: flicker 1.5s infinite alternate;     
+}
+
+h1::-moz-selection {
+  background-color: var(--neon-border-color);
+  color: var(--neon-text-color);
+}
+
+h1::selection {
+  background-color: var(--neon-border-color);
+  color: var(--neon-text-color);
+}
+
+h1:focus {
+  outline: none;
+}
+
+:root {
+  /* Base font size */
+  font-size: 10px;   
+  
+  /* Set neon color */
+  --neon-text-color: rgb(255, 0, 0);
+  --neon-border-color: rgb(20, 169, 255);
+}
+
+/* Animate neon flicker */
+@keyframes flicker {
+    
+    0%, 19%, 21%, 23%, 25%, 54%, 56%, 100% {
+      
+        text-shadow:
+            -0.2rem -0.2rem 1rem #fff,
+            0.2rem 0.2rem 1rem #fff,
+            0 0 2rem var(--neon-text-color),
+            0 0 4rem var(--neon-text-color),
+            0 0 6rem var(--neon-text-color),
+            0 0 8rem var(--neon-text-color),
+            0 0 10rem var(--neon-text-color);
+        
+        box-shadow:
+            0 0 .5rem #fff,
+            inset 0 0 .5rem #fff,
+            0 0 2rem var(--neon-border-color),
+            inset 0 0 2rem var(--neon-border-color),
+            0 0 4rem var(--neon-border-color),
+            inset 0 0 4rem var(--neon-border-color);        
+    }
+    
+    20%, 24%, 55% {        
+        text-shadow: none;
+        box-shadow: none;
+    }    
+}
+
+
+h2 {
+  font-family: 'Titillium Web', sans-serif;
+  font-size: 20px;
+  color:#fff;
+  margin-left: 10px;
+}
+
+.form-container {
+  background-color: rgba(190, 219, 252, 0.342);
+  margin: auto;
+  width: 50%;
+  border-radius: 20px;
+}
+
+.summary-container {
+  font-size: 40px;
+  margin:auto;
+  padding: 10px;
+  text-align: center;
+  align-items: center;
+  border-radius: 50px;
+}
+
+.summary-container button {
+  width: 50%;
+  font-size:20px;
+  background-color: #860525;
+  color: #fff;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.summary-image {
+  max-width: 500px;
+  max-height: 300px;
+  margin: 30px;
+}
+
+form {
+  display:flex;
+  flex-direction: column;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right:30px;
+  padding-left: 30px;
```

Comment:
I learned from Maks in one of his last code reviews that you always start with "top" and then go round like the clock, so top, left, bottom, right.

Reply:
nice, missed that! thank you!